unicore/distribute/api/repos.py

```diff
@@ -73,7 +104,7 @@ def post(self):
                     'url': self.request.route_url('repositoryresource',
                                                   name=name)
                 }))
-        return format_diffindex(changes)
+        return ''
```

Comment:
Oh, hm, that's surprising. Yeah, let's return the changes from the `workspace.pull()` if needed.